dist/dfu.js

```diff
@@ -382,7 +382,8 @@
                         case OPCODE.VALIDATE_FIRMWARE:
                             log('complete, reset...');
 
-                            controlChar.service.device.addEventListener("gattserverdisconnected", function() {
+                            server.device.addEventListener("gattserverdisconnected", function() {
```

Comment:
@beaufortfrancois 

maybe you know if this is a bug or just yet to be implemented?